.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: enhancement, triage
+assignees: ""
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is.
```

Comment:
Do we want to encourage the use of GH discussions for more complex or controversial feature requests? Or perhaps we can just keep it like this and move the issue to a discussion if it feels like it is going to lead to wide ranging discussion?

Reply:
+1 to 

> perhaps we can just keep it like this and move the issue to a discussion if it feels like it is going to lead to wide ranging discussion?

(and we can improve the templates when the need arises)